server/cmwell-dc/src/pack/logback.xml

```diff
@@ -210,6 +233,10 @@
         <appender-ref ref="ASYNC_YELLOW_LOG" />
     </logger>
 
+    <logger name="bad-data" additivity="true">
```

Comment:
Same argument as before. Do you want the log lines also on the main log file?

Reply:
Done